### DISCUSSION TITLE
Adding Schlage Encode support via Ring App via Amazon Key

### DISCUSSION THREAD

**Discussion by GarrickVance** [dgreif/ring#654] [Ideas]
This plugin has been fantastic!  Currently the Schlage Encode wifi locks are not Homekit enabled, but do work now within the Ring app (via Amazon Key connection) and adds a lock/unlock capability when on live view. (https://support.ring.com/hc/en-us/articles/360027029751-Works-With-Ring-Schlage-Encode-Smart-Wifi-Deadbolt-Information)

Would it be possible to extend and add Schlage Encode capability through the Ring plugin here?  This seems like the most promising solution since Schlage seems to have dropped homekit with the newer locks and the HomeBridge plugin for Schlage is only one older lock.

**Comment by dgreif** [comment]
See https://github.com/dgreif/ring/issues/384#issuecomment-660169540. I looked into it, but it's not in the scope of the Ring plugin unfortunately.
